reversers/database-synchroniser/plugins/org.obeonetwork.dsl.database.liquibasegen/src/org/obeonetwork/dsl/database/liquibasegen/service/GenServices.java

```diff
@@ -0,0 +1,130 @@
+package org.obeonetwork.dsl.database.liquibasegen.service;
+
+import java.util.List;
+import java.util.Optional;
+
+import org.eclipse.emf.compare.Comparison;
+import org.eclipse.emf.ecore.EObject;
+import org.obeonetwork.dsl.database.Schema;
+import org.obeonetwork.dsl.database.Table;
+import org.obeonetwork.dsl.database.dbevolution.DBDiff;
+import org.obeonetwork.dsl.database.gen.common.services.TypesServices;
+import org.obeonetwork.dsl.database.gen.common.services.UtilServices;
+import org.obeonetwork.dsl.typeslibrary.TypeInstance;
+
+/**
+ * Class that provides service for the Liquibase generation
+ * 
+ * <p>
+ * It intentionally wrap other service classes to clearly identified which one a
+ * required for the Liquibase generation. It may help in a future refactoring
+ * when slipping the Liquibase generation in another plugin
+ * </p>
+ * 
+ * @author adaussy
+ *
+ */
+public class GenServices {
+
+	private static final String EMPTY = "";
+
+	private TypesServices typeService = new TypesServices();
+
+	private UtilServices utilService = new UtilServices();
+
+	/**
+	 * Gets the qualified name of a {@link Table} ($schemaName.$tableName)
+	 * 
+	 * @param table a {@link Table}
+	 * @return a String
+	 */
+	public String fullName(Table table) {
```

Comment:
Shouldn't this method named genFullName to be consistent with the others ?

Reply:
Yes. It was the named used in AQL so I used the same bu tyour are right